core/pkg/launch/job_builder.go

```diff
@@ -181,18 +191,39 @@ func MakeArtifactNameSafe(name string) string {
 
 }
 
-func NewJobBuilder(settings *service.Settings, logger *observability.CoreLogger) *JobBuilder {
+func NewJobBuilder(settings *service.Settings, logger *observability.CoreLogger, verbose bool) *JobBuilder {
 	jobBuilder := JobBuilder{
 		settings:              settings,
 		isNotebookRun:         settings.GetXJupyter().GetValue(),
 		logger:                logger,
 		Disable:               settings.GetDisableJobCreation().GetValue(),
 		wandbConfigParameters: newWandbConfigParameters(),
 		saveShapeToMetadata:   false,
+		verbose:               verbose,
 	}
 	return &jobBuilder
 }
 
+func (j *JobBuilder) logIfVerbose(msg string, level LogLevel) {
+
+	switch {
+	case level == Log:
+		j.logger.Info(msg)
+	case level == Warn:
+		j.logger.Warn(msg)
+	case level == Error:
+		j.logger.Error(msg)
+	default:
+		j.logger.Info(msg)
+	}
+
+	if j.verbose {
+		// TODO: we should pass messages back to the user
+		// and print them there, instead of printing them here
+		fmt.Println(msg)
+	}
```

Comment:
i think we can maybe clean up our logger to be able to print to multiple writers... 
we only set it up currently in the beginning, but it would be nice to have this wrapper that allows to dynamically change it for each object.
It is in our plans when we re-do the loggers in a few weeks